e2e-tests/playwright/tests/functional/channels/edit_file_attachments/edit_file_attachment.spec.ts

```diff
@@ -0,0 +1,372 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {test} from '@e2e-support/test_fixture';
+import pages from '@e2e-support/ui/pages';
+import {expect} from '@playwright/test';
+import {duration, wait} from '@e2e-support/util';
+
+test('MM-T5654_1 should be able to edit post message', async ({pw}) => {
+    test.setTimeout(120000);
+
+    const originalMessage = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit';
+
+    const {user} = await pw.initSetup();
+    const {page} = await pw.testBrowser.login(user);
+    const channelPage = new pages.ChannelsPage(page);
+
+    await channelPage.goto();
+    await channelPage.toBeVisible();
+    await channelPage.centerView.postCreate.postMessage(originalMessage);
+
+    const post = await channelPage.centerView.getLastPost();
+    await post.toBeVisible();
+    await post.hover();
+    await post.postMenu.toBeVisible();
+
+    // open the dot menu
+    await post.postMenu.dotMenuButton.click();
+    await channelPage.postDotMenu.toBeVisible();
+    await channelPage.postDotMenu.editMenuItem.click();
+    await channelPage.centerView.postEdit.toBeVisible();
+    await channelPage.centerView.postEdit.writeMessage('Edited message');
+    await channelPage.centerView.postEdit.sendMessage();
+
+    const updatedPost = await channelPage.centerView.getLastPost();
+    await updatedPost.toBeVisible();
+    await updatedPost.toContainText('Edited message');
+});
+
+test('MM-T5655_3 should be able to edit post message in RHS', async ({pw}) => {
```

Comment:
There are 3 automated tests here but the [written test case](https://github.com/mattermost/mattermost-test-management/blob/main/data/test-cases/channels/edited-post/MM-T5655.md) has one only. Maybe need to update the written test case?